packages/upstream-protobuf/bin/upstream-inject-feature-defaults.mjs

```diff
@@ -45,7 +46,11 @@ async function main(args) {
   );
   for (const path of positionals) {
     const content = readFileSync(path, "utf-8");
-    const r = inject(content, ` "${defaults.toString("base64url")}" `);
+    const base64String = defaults.toString("base64url");
+    const buffer = protoBase64.dec(base64String);
+    const featureSetDefaults = FeatureSetDefaults.fromBinary(buffer);
```

Comment:
Depending on our own runtime can be an issue here, because we need this package to bootstrap it 😓 

I'm not completely sure it'll be problem in practice now or in the future, but I'm sure that not using our own runtime within this helper package avoids the problem 100% 🙂

Reply:
Yep I did think about it, but as we only depend on some google types, it didn't seem to be an issue. However I do agree it's best to avoid it :)